Orange/widgets/unsupervised/owkmeans.py

```diff
@@ -128,6 +128,7 @@ class Warning(widget.OWWidget.Warning):
     max_iterations = Setting(300)
     n_init = Setting(10)
     smart_init = Setting(0)  # KMeans++
+    random_state = Setting(0)
```

Comment:
You make this a setting, but it's not tunable?

Reply:
Now that you point it out, it looks like at least a comment here would be appropriate.
But to explain, we discussed how to allow users to change the random seed and not yet come to the final conclusion. Maybe a common option in Orange's settings or per widget buttons to change the seed... But everyone mostly agrees that widgets should by default produce reproducible results, so we want to start fixing the seeds and then add the option to change that if/where we decide it is best (proposals should probably go into #3103).
For now I don't mind if random_state is a constant or a setting (with a comment to avoid confusion).

Reply:
I made it a constant.